env/magpie.env.example

```diff
@@ -21,10 +23,19 @@ PHOENIX_PORT=8443
 PHOENIX_PUSH=false
 TWITCHER_PROTECTED_PATH=/ows/proxy
 
-# Magpie Tests
+## === Magpie Tests ===
+# below version defines the override value for test
+# when using tes safeguards for different versions, new tests that require 'future' tags yet to be defined can
```

Comment:
Small typo : missing 't' from test in "when using tes"

Reply:
nice catch ;)